src/main/java/edu/ucdavis/dss/ipa/api/components/assignment/AssignmentViewTeachingAssignmentController.java

```diff
@@ -213,11 +213,8 @@ public TeachingAssignment updateTeachingAssignment(@PathVariable long teachingAs
                 sectionGroup = sectionGroupService.save(sectionGroup);
             }
 
-            // Associate teachingAssignment to the newly created sectionGroup and remove the suggested course metadata
+            // Associate teachingAssignment to the newly created sectionGroup
             originalTeachingAssignment.setSectionGroup(sectionGroup);
-            originalTeachingAssignment.setSuggestedEffectiveTermCode(null);
-            originalTeachingAssignment.setSuggestedSubjectCode(null);
-            originalTeachingAssignment.setSuggestedCourseNumber(null);
             originalTeachingAssignment.setApproved(teachingAssignment.isApproved());
```

Comment:
I'm puzzled, how does this connect to the card? and should this logic be deleted? It seems right, unless I'm missing something.

It looks like that whole if block is applying to teachingAssignments where it has been approved, and was tied to a suggested course,

It then creates the ‘suggested course’, and ties it to the preference -

and at that point, sets the ‘suggested’ stuff to null as this is now a real course, not a suggested course.

Which sounds fuzzily correct. a preference can be tied to:
- a sectionGroup
- a non-sectionGroup thing (buyout, courseRelease,etc)
- a suggested course

but it should only connect to one of those at a time

Reply:
I went ahead and added the logic back to remove the suggested course metadata and updated it to remove the suggestedTitle field as well.